src/frequenz/dispatch/_dispatcher.py

```diff
@@ -192,10 +195,32 @@ def __init__(
         self._empty_event = Event()
         self._empty_event.set()
 
+    @override
     def start(self) -> None:
         """Start the local dispatch service."""
         self._bg_service.start()
 
+    @property
+    @override
+    def is_running(self) -> bool:
+        """Whether the local dispatch service is running."""
+        return self._bg_service.is_running
+
+    @override
+    async def wait(self) -> None:
+        """Wait until all actor dispatches are stopped."""
+        await self._empty_event.wait()
```

Comment:
I would `await self._bg_service` too here (probably using `gather` to do it in parallel).